examples/dir.roc

```diff
@@ -27,6 +27,13 @@ main =
     expect
         createChildShouldSucceed == Ok {}
 
+    # List the contents of a directory
+    _ <- 
+        Path.fromStr "a"
+        |> Dir.list 
+        |> Task.onErr \_ -> crash "Failed to list directory"
+        |> Task.await
```

Comment:
Could you check if the returned list matches the expected ["a"]?

Reply:
Done